docs/source/user/parallel.rst

```diff
@@ -67,8 +71,14 @@ Another experimental feature of this module is support for explicit parallel
 loops. One can use Numba's ``prange`` instead of ``range`` to specify that a
 loop can be parallelized. The user is required to make sure that the loop does
 not have cross iteration dependencies except the supported reductions.
-Currently, reductions on scalar values are supported and are inferred from
-in-place operations. The example below demonstrates a parallel loop with a
+
+A reductions is inferred automatically if a variable is updated by a binary
+function/operator using its previous value in the loop body. The initial value
+of the reduction is inferred automatically for ``+=`` and ``*=`` operators.
```

Comment:
Subtraction is not supported in reductions since it is not commutative. Supporting it could lead to substantial complexity. The user can easily use summation and subtract the final result.

Reply:
Ah yes, that's a fair point. Let's leave it.